ps2/pavla ps2.R

```diff
@@ -0,0 +1,213 @@
+##############################################################################
+# Problem Set 2
+##############################################################################
+
+##############################################################################
+# Instructions
+##############################################################################
+# 1) You should submit the .R script with the solution
+# 2) Make sure to comment wherever it is needed
+# 3) Every student should submit his\hers own copy
+
+
+##############################################################################
+# Problem 1 (70%)
+##############################################################################
+
+# You'll be working with adult.data . Here's a brief description of the 
+# variables:
+# age: continuous.
+# workclass: Private, Self-emp-not-inc, Self-emp-inc, Federal-gov, Local-gov, State-gov, 
+#            Without-pay, Never-worked.
+# fnlwgt: continuous.
+# education: Bachelors, Some-college, 11th, HS-grad, Prof-school, Assoc-acdm, Assoc-voc,
+#            9th, 7th-8th, 12th, Masters, 1st-4th, 10th, Doctorate, 5th-6th, Preschool.
+# education-num: continuous.
+# marital-status: Married-civ-spouse, Divorced, Never-married, Separated, Widowed, Married-spouse-absent, Married-AF-spouse.
+# occupation: Tech-support, Craft-repair, Other-service, Sales, Exec-managerial, Prof-specialty, Handlers-cleaners, Machine-op-inspct, Adm-clerical, Farming-fishing, Transport-moving, Priv-house-serv, Protective-serv, Armed-Forces.
+# relationship: Wife, Own-child, Husband, Not-in-family, Other-relative, Unmarried.
+# race: White, Asian-Pac-Islander, Amer-Indian-Eskimo, Other, Black.
+# sex: Female, Male.
+# capital-gain: continuous.
+# capital-loss: continuous.
+# hours-per-week: continuous.
+# native-country: United-States, Cambodia, England, Puerto-Rico, Canada, Germany, Outlying-US(Guam-USVI-etc), India, Japan, Greece, South, China, Cuba, Iran, Honduras, Philippines, Italy, Poland, Jamaica, Vietnam, Mexico, Portugal, Ireland, France, Dominican-Republic, Laos, Ecuador, Taiwan, Haiti, Columbia, Hungary, Guatemala, Nicaragua, Scotland, Thailand, Yugoslavia, El-Salvador, Trinadad&Tobago, Peru, Hong, Holand-Netherlands.
+# salary: >50K,<=50K
+
+# Further description is available in adult.names file
+
+# 1) read the data, make sure columns are read in the right format, i.e. character, double, etc.
+
+library(magrittr)
+library(tidyverse)
+library(purrr)
+library(foreign)
+library(dplyr)
+library(stringr)
+
+adult <- read.csv("adult.data", header=F)
+as.data.frame(adult)
+
+# 2) How many men and women are represented in the dataset? 
+
+print(unique(adult$V10))
+# There are some blank spaces
+
+female <- adult %>%
+  filter(V10 == " Female") %>%
+  nrow()
+
+male <- adult %>%
+  filter(V10 == " Male") %>%
+  nrow()
+
```

Comment:
correct, but it would be a lot easier to select(Sex) %>% summarize(number  = count())
smth of that sort. Single line  -> easier to understand

the result is correct though